app/graphql/types/plans/object.rb

```diff
@@ -41,7 +41,10 @@ def charges_count
       end
 
       def customers_count
-        object.subscriptions.active.select(:customer_id).distinct.count
+        customers_count = object.subscriptions.active.select(:customer_id).distinct.count
+        return customers_count unless object.children
+
+        customers_count + object.children.sum { |c| c.subscriptions.active.select(:customer_id).distinct.count }
```

Comment:
Do we have a way to do it in a single DB query? Here we will be doing one query per child plan :/